client/components/SolveExercise/StatusBanner/StatusBanner.tsx

```diff
@@ -1,11 +1,11 @@
 import React, { useState } from 'react';
-import { SUBMISSION_STATUS } from '~/../shared/SharedConstants';
+import { SubmissionStatus } from '~/../shared/SharedConstants';
 import Button from '~/components/Button';
 
 import styles from './StatusBanner.module.scss';
 
 interface Props {
-  status: string; // SUBMISSION_STATUS
+  status: string; // SubmissionStatus
```

Comment:
The comment was here to indicate this string is actually referring to the submission status. 😅

Now that we have this Interface let's actually add it here:

`status: SubmissionStatus`

Reply:
This was a global rename 😂

Reply:
Got it. It's fine to leave it as is, it's not part of this issue scope